cypress/support/helper.js

```diff
@@ -22,7 +22,10 @@ export function visitComponentUrl(
   prefix = "",
   env = "story"
 ) {
-  cy.visit(prepareUrl(component, suffix, iFrameOnly, prefix, env));
+  const spy = cy.spy();
+  cy.visit(prepareUrl(component, suffix, iFrameOnly, prefix, env), {
+    onBeforeLoad: spy,
```

Comment:
Why?

Reply:
because it wasn't working for both (`BDD`(actual approach) and `component testing`(future approach))